src/charm.py

```diff
@@ -55,15 +57,23 @@
     service_running,
     service_stop,
 )
-from charms.prometheus_k8s.v0.prometheus_scrape import MetricsEndpointAggregator
+from charms.prometheus_k8s.v0.prometheus_scrape import (
+    MetricsEndpointAggregator,
+    _type_convert_stored,
```

Comment:
This method seems to be useful outside of its library. I wonder whether we should eventually make it not "private" then.